webdriver/tests/classic/perform_actions/wheel.py

```diff
@@ -55,6 +55,12 @@ def test_scroll_iframe(session, test_actions_scroll_page, wheel_chain):
 
     wheel_chain.scroll(0, 0, 5, 10, origin=target).perform()
 
+    # Chrome requires some time to process the event from the iframe, so we wait for it.
+    while True:
```

Comment:
I believe this throws

E     File "/home/test/web-platform-tests/webdriver/tests/classic/perform_actions/wheel.py", line 60
E       events = await get_events(bidi_session, top_context["context"])
E                ^
E   SyntaxError: 'await' outside async function

Reply:
Indeed. Fixed. Thanks!